### PR TITLE
fix: Changing some language around fees

### PR DESCRIPTION
**Pull request by mattbspector**
## Description
Per product + design, changing some language related to fees

### CODE REVIEW

File: src/components/swap/GasBreakdownTooltip.tsx [hunk 1]
```diff
@@ -69,12 +69,10 @@ export function GasBreakdownTooltip({ trade, hideUniswapXDescription }: GasBreak
...
 }
 
-function NetworkFeesDescription({ native }: { native: Currency }) {
+function NetworkCostDescription({ native }: { native: Currency }) {
   return (
     <ThemedText.LabelMicro>
-      <Trans>
-        The fee paid to the Ethereum network to process your transaction. This must be paid in {native.symbol}.
-      </Trans>{' '}
+      <Trans>The cost to transact on the Ethereum network. This must be paid in {native.symbol}.</Trans>{' '}
```
Comment (cbachmeier) [inline src/components/swap/GasBreakdownTooltip.tsx:75]: blocking: we should update this to the name of the network, in the screenshot below I would be paying the fee to the Polygon network not Ethereum
Screenshot 2023-10-18 at 12 14 17 PM

non-blocking: Also I would personally word this as `The cost of this transaction on the X network`. The current wording makes it seem like this is a flat fee per network

**Review by cbachmeier — CHANGES_REQUESTED** [review]

File: src/components/swap/GasBreakdownTooltip.tsx (hunk 1 above)
Comment (mattbspector) [reply src/components/swap/GasBreakdownTooltip.tsx:75]: @bryankuuu just want to get your 👀 here as well to sign off on the above

**Comment by gnewfield** [comment]
Looks like the unit tests in `src/components/swap/SwapModalFooter.test.tsx` and `src/components/swap/SwapLineItem.test.tsx`, as well as the corresponding snapshots, are failing because of the new message. We should update these.

**Review by gnewfield — CHANGES_REQUESTED** [review]
Looks like the unit tests in `src/components/swap/SwapModalFooter.test.tsx` and `src/components/swap/SwapLineItem.test.tsx`, as well as the corresponding snapshots, are failing because of the new message. We should update these.

File: src/components/swap/GasBreakdownTooltip.tsx (hunk 1 above)
Comment (infredible) [reply src/components/swap/GasBreakdownTooltip.tsx:75]: Agree, this the name should be variable to match the chain.

I hear your suggestion, but the intent behind the copy was to more clearly attribute the cost to the chain and disambiguate from ourselves. All of the values here are variable, so I'm personally less concerned with it being mistaken as a flat fee. wdyt @bryankuuu ?

Comment (bryankuuu) [reply src/components/swap/GasBreakdownTooltip.tsx:75]: Let's go with this
"Network cost is paid in Y to the X network in order to transact. Learn more"

Thank you!

**Comment by mattbspector** [comment]
Screenshot 2023-10-30 at 12 13 14 PM


@cbachmeier @bryankuuu @infredible pushed up the requested change here

**Comment by mattbspector** [comment]
@gnewfield just pushed up fixes for those tests 🙏

File: src/components/swap/GasBreakdownTooltip.tsx (hunk 1 above)
Comment (mattbspector) [reply src/components/swap/GasBreakdownTooltip.tsx]: pushed this change!

File: src/components/swap/GasBreakdownTooltip.tsx [hunk 2]
```diff
@@ -69,11 +70,13 @@ export function GasBreakdownTooltip({ trade, hideUniswapXDescription }: GasBreak
...
 
-function NetworkFeesDescription({ native }: { native: Currency }) {
+function NetworkCostDescription({ native }: { native: Currency }) {
+  const chainName = chainIdToBackendName(native.chainId)
+
   return (
     <ThemedText.LabelMicro>
       <Trans>
-        The fee paid to the Ethereum network to process your transaction. This must be paid in {native.symbol}.
+        Network cost is paid in {native.symbol} on the {chainName} network in order to transact.
```
Comment (natew) [inline src/components/swap/GasBreakdownTooltip.tsx:79]: I actually don't know how this works but just curious - i imagine `<Trans>` relies on the inner string contents to do the translation, but here we have multiple slots that are filled in eg `{native.symbol}`, does Trans still work?

Comment (natew) [reply src/components/swap/GasBreakdownTooltip.tsx:79]: not sure who is best suited to answer maybe @zzmp

Comment (tinaszheng) [reply src/components/swap/GasBreakdownTooltip.tsx:79]: so lingui is smart enough to extrapolate variables inside jsx correctly, see the example on their home page :D 
https://lingui.dev/introduction

**Review by cbachmeier — APPROVED** [review]

**Review by gnewfield — APPROVED** [review]

**Pull request merged**